server/test/unit/sensorAlertsTest/handleAlertTest.js

```diff
@@ -195,11 +206,13 @@ describe('sensorAlerts.js unit tests: handleAlert', () => {
       sandbox.stub(db, 'createSession')
       sandbox.stub(db, 'saveSession')
 
-      await sensorAlerts.handleAlert(this.location, alertType)
+      await handleAlert(this.location, alertType, '{"numberOfAlerts": 1}')
```

Comment:
Instead of 1 (or as a new test case in addition to 1), it would be worth it to try `numberOfAlertsToAcceptResetRequest-1` to make sure that that boundary also does not give the option to reset.

Reply:
Good point! I'll use `numberOfAlertsToAcceptResetRequest - 1`.